packages/css-framework/src/components/_list.scss

```diff
@@ -0,0 +1,63 @@
+@use "../abstracts/functions" as f;
+
+.rn-list {
+  border: f.spacing("px") solid f.color("neutral", "100");
```

Comment:
I'm not familiar with this f.spacing `px` usage - what does this do?

Reply:
It is for `1px`:

> as everything else is set in rems having a single pixel value is often quite useful